app/transformers/api/v3/transformer.rb

```diff
@@ -1,7 +1,19 @@
 class Api::V3::Transformer
   class << self
+    def redirect_to_deduped_patient(attributes)
+      # NOTE: Move this to a different layer if/when this becomes more complex
+      deduped_record = DeduplicationLog.find_by_deleted_record_id(attributes["id"])&.deduped_record
+      deduped_patient = DeduplicationLog.find_by_deleted_record_id(attributes["patient_id"])&.deduped_record
```

Comment:
How does this clause work when the record itself is a patient? Does it behave gracefully?